plugin/al/plugin/AL_USDMayaTestPlugin/py/CMakeLists.txt

```diff
@@ -35,10 +35,28 @@ add_test(
                                       "
 )
 
-set_property(TEST ${TEST_NAME} APPEND PROPERTY ENVIRONMENT 
+set_property(TEST TestUSDMayaPython APPEND PROPERTY ENVIRONMENT 
     "PYTHONPATH=${pythonPath}"
     "PATH=${path}"
     "MAYA_PLUG_IN_PATH=${mayaPluginPath}"
     "PXR_PLUGINPATH_NAME=${CMAKE_INSTALL_PREFIX}/lib/usd"
     "MAYA_NO_STANDALONE_ATEXIT=1"
 )
+
+add_test(
+    NAME TestUSDMayaPythonModuleOnly
+    WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+    # Note that, since this test does NOT call maya.standalone.initialize, we can
+    # just let unittest.main call sys.exit "normally"
+    COMMAND ${MAYA_PY_EXECUTABLE} -c "import sys;from unittest import main; \
+                                      import testUSDMayaPythonModuleOnly; testProg = main(module=testUSDMayaPythonModuleOnly); \
+                                      "
```

Comment:
Make exit with proper status reflecting test results. 
```
sys.exit(not testProg.result.wasSuccessful()); \
```

Reply:
unittest.main automatically does precisely what you wrote.

The only reason we needed to break it up - ie, use `main(exit=False)`, capture the testProg, get it's result and then call `sys.exit` explicitly, was because we needed to call some code inbetween the test finishing, and calling `sys.exit` - namely, `maya.standalone.uninitialize`.

The comment I added, about not calling maya.standalone.initialize, was meant to explain that.  I can expand that a bit, if you'd like, stating more explicitly what I've just explained.

Or, if you still prefer making the boilerplate be as similar as possible, I can add in the sys.exit / testProg / exit=False stuff, though it's not strictly necessary.

Reply:
Oops - I guess, if we're going the simpler approach, I should also change

```
testProg = main(...)
```

to be just

```
main(...)
```

Reply:
Didn't notice that one! Making it main(...) sounds like the right thing to do.